lib/src/snippet/kotlin/com/lemonappdev/konsist/LibrarySnippets.kt

```diff
@@ -12,6 +16,35 @@ class LibrarySnippets {
             .assert { it.hasKDoc }
     }
 
+    fun `every function with parameters has a param tags`() {
+        Konsist.scopeFromPackage("..api..")
+            .functions(includeNested = true)
+            .assert { it.hasValidParameterKDoc() }
+    }
+
+    fun `every function with return value has a return tag`() {
+        Konsist.scopeFromPackage("..api..")
+            .functions(includeNested = true)
+            .assert { it.hasValidReturnTypeKDoc() }
+    }
+
```

Comment:
I added two options of `every extension has a receiver tag` test. I don't know which version is better.
Is any better way to filter `declarations`  by several providers (in this case we need all declarations which are KoReceiverTypeProvider and KoKDocProvider instances)?

Reply:
Let's leave only option 2